src/ralph/account/management/commands/ldap_sync.py

```diff
@@ -22,6 +23,20 @@
     ldap_module_exists = False
 
 
+def _truncate_surname(ldap_dict):
+    """
+    Truncate user's surname when it's longer then default django value, which is
+    30 chars.
+    """
+    if 'sn' in ldap_dict:
+        valid_sns = []
+        for surname in ldap_dict['sn']:
+            max_length = User._meta.get_field('last_name').max_length
```

Comment:
Why are you retrieving this in every iteration?

Maybe you could turn the whole loop into a comprehension?